Spigot-Server-Patches/0555-Zero-pad-chunks-before-saving-them-to-the-disk.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Toon Schoenmakers <nighteyes1993@gmail.com>
+Date: Mon, 24 Aug 2020 15:12:06 +0200
+Subject: [PATCH] Zero pad chunks before saving them to the disk
+
+This makes the unused data inside region files more repeatable.
+Which will result in better compression in case the entire world directory
+is compressed (like in backups). Which will naturally result in smaller backups.
+
+diff --git a/src/main/java/net/minecraft/server/RegionFile.java b/src/main/java/net/minecraft/server/RegionFile.java
+index 939b51826681a7648ad3ea2512c20082193e6a01..de2a2599b651597ef81febe83bfcba3bc760400d 100644
+--- a/src/main/java/net/minecraft/server/RegionFile.java
++++ b/src/main/java/net/minecraft/server/RegionFile.java
+@@ -497,6 +497,8 @@ public class RegionFile implements AutoCloseable {
+         }
+ 
+     }
++
++    private final byte[] zero = new byte[4096];
```

Comment:
This needs to be a Direct Memory buffer, and static, as otherwise the JVM copies memory every single write